solutions/silver/cf-51C.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: cf-51C
+source: CF
+title: Three Base Stations
+author: KJ Karaisz
```

Comment:
lol why the name change

Reply:
My real name hasn't appeared on anything "professional-looking" before. I thought about it and this makes more sense to use.